source/_docs/modules-plugins-known-issues.md

```diff
@@ -339,6 +339,12 @@ For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_nam
 **Solution**: See [Caching: Advanced Topics](/docs/caching-advanced-topics/) for details on how to bypass the platform page cache.
 <hr>
 
+### [Force Login](https://wordpress.org/plugins/wp-force-login/){.external}
+**Issue**: This plugin appends a port number using `$_SERVER['SERVER_PORT']` at the end of the URL when the user logs in to the site.
```

Comment:
Is it always the same port number? What number is it? What error does this cause, if any?

Reply:
It depends on what the port number the appserver is using. You can check that port number on the debug pannel of the site
![children s hospital of philadelphia pantheon dashboard 2018-07-28 09-48-37](https://user-images.githubusercontent.com/1130280/43351867-7e5d6f24-924b-11e8-91a7-a4e0e92dfada.png)

Reply:
@alexfornuto It's not the same port. The port number is random based on the port number used by the appserver as shown in the image. The result is a white page.

You can test it here: http://force-login-jts-wp.pantheonsite.io/hello-world
user: josephpantheon
pass: WbjgZ(Bt#hVKq3zDUT)uekR2